ci/clean.py

```diff
@@ -169,13 +175,17 @@ def clean_orphaned_objects(
 
         continuation_token = res.get('NextContinuationToken')
 
+        print('Got next chunk from S3')
         object_keys = {obj_dict['Key'] for obj_dict in res['Contents']}
 
         # determine those keys that are no longer referenced by any manifest
         loose_object_keys = object_keys - all_object_keys
 
         if dry_run:
-            f'would delete {len(loose_object_keys)=} unreferenced objs'
+            print(f'would delete {len(loose_object_keys)=} unreferenced objs:')
```

Comment:
nit: use logger

Reply:
Agreed but this is not my code. Not sure if print() is okay for a CLI command.
Suggestion: Use logger across whole project?